jobs/config.json

```diff
@@ -3082,8 +3082,8 @@
       "--check-leaked-resources=true"
     ],
     "scenario": "kubernetes_e2e",
-    "sigOwners": [
-      "UNKNOWN"
+   "sigOwners": [
```

Comment:
the indention is wrong, and in other places. run jobs/config_sort.py will fix those.